### ISSUE TITLE
CreateChain route does not create address for joining user

### ISSUE THREAD

**Issue by jnaviask** [hicommonwealth/commonwealth#4205]
## Description

Currently, createChain uses a `createRole` to assign admin to an address, but it always uses an address on another community, violating the contract of 1 address row per community. We need to ensure createChain makes a new address row so that we can assign admin properly.

## Acceptance Criteria
CreateChain route should set the creating user as joined + admin role, if possible (i.e. if they have a matching chain-base address).

**Comment by alexyoung23j** [comment]
@kurtisassad BD QA has just verified that this was not actually fixed.

**Comment by kurtisassad** [comment]
@alexyoung23j I noticed a new bug where when you create a community the "create thread" button is greyed out until you refresh. I will work on fixing this.

In regards to the not creating address, I can't seem to recreate this. Tried creating communities for starter community, erc20, erc721. They seem to work fine (admin address in db). Is there a specific sequence I can perform to reproduce this?
